components/blitz/src/ome/services/blitz/repo/ManagedImportProcessI.java

```diff
@@ -24,23 +24,25 @@
 import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutionException;
 
+import org.perf4j.StopWatch;
+import org.perf4j.slf4j.Slf4JStopWatch;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
 import org.springframework.aop.framework.Advised;
 
 import com.google.common.cache.Cache;
 import com.google.common.cache.CacheBuilder;
 
 import Ice.Current;
-
 import ome.services.blitz.impl.AbstractCloseableAmdServant;
 import ome.services.blitz.impl.ServiceFactoryI;
 import ome.services.blitz.repo.PublicRepositoryI.AMD_submit;
 import ome.services.blitz.repo.path.FsFile;
 import ome.services.blitz.util.ServiceFactoryAware;
-
```

Comment:
I quite like these spaces but if you intended to remove them that's fine.

Reply:
No, I assume Eclipse stole them.